### ISSUE TITLE
Linux Compilation

### ISSUE THREAD

**Issue by ComProf** [AshamaneProject/AshamaneCore#230]
many errors when compile on linux in /src/server/scripts/Commands/cs_export.cpp
Latest commit ef978be
Branch: master

**Comment by DuelistRag3** [comment]
A temp workaround for it is to just delete the cs_export.cpp and re run cmake.
Of course its not a long term solution

**Comment by DuelistRag3** [comment]
This happens because there are hardcoded windows requierements in the cs_export.cpp

**Comment by ComProf** [comment]
Yes I`ll do it, but it`s arena... So arena didn`t work`s without it? (Arena commands,cache)

**Comment by DuelistRag3** [comment]
Not 100 % sure, but don't think so, because it's only a in game command
